installer/windows/build.bat

```diff
@@ -1,12 +1,17 @@
-rem set the TAG env var from the release dir
-for /f %%i in ('dir /B release') do set TAG=%%i
+@echo off
+rem build msi installer for each release directory
+for /d %%d in (release\*) do (
```

Comment:
maybe we can just add two params for `storagenode.exe` and `storagenode-updater.exe`? and if we will want to have script that is building for each release dir we can use this build script. it will be easier to use it manually from command line.

Reply:
Similar to the reply below on `installrelease.bat`:
>The reason for the for loop is just to get into the release directory, whose name we don't know. Perhaps there's a more concise way to do this.`

In this case it's possible to do this for all release directories so I just didn't stop the loop like in `installrelease.bat`. There may not be much value in this.